velox/functions/sparksql/DateTimeFunctions.h

```diff
@@ -408,4 +408,70 @@ struct DayOfYearFunction {
     result = getDayOfYear(getDateTime(date));
   }
 };
+
+static const folly::F14FastMap<std::string, int8_t> kDayOfWeekNames{
```

Comment:
No global `static` in header, move this to `.cpp` file

Reply:
@Yuhta done

Reply:
@Yuhta I have moved kDayOfWeekNames to TimeUtils, where it is now alongside some other time-related constants. Could you please review this?